libraries/transformer/src/androidTest/java/androidx/media3/transformer/mh/HdrEditingTest.java

```diff
@@ -139,6 +139,41 @@ public void export_transmuxHlg10File() throws Exception {
     assertThat(actualColorTransfer).isEqualTo(C.COLOR_TRANSFER_HLG);
   }
 
+  @Test
+  public void export_transmuxDolbyVisionFile() throws Exception {
```

Comment:
Could you give me more hint about it?
1. I added the Dolby Vision test case here because there are export_transmuxHdr10File() and export_transmuxHlg10File() test cases in this file. I did the similar thing in my test case implementation.
2. In MediaItemExportTest, I found some transmux test cases but I didn't find corresponding test cases for HLG and HDR10. Do I need to add test case for Dolby Vision in this file?

Reply:
I think these tests needs to run on real device because in `Robolectric` tests, `FrameworkMuxer` does not write to the actual output file and we are doing validations on the actual output file. So we should keep the test case here only. Thanks!